cta_names_mappings.md

```diff
@@ -1,169 +1,621 @@
 * Organisation = [Unit 26165 of the 85th Main Special Service Center](https://www.cisa.gov/uscert/ncas/alerts/aa22-110a)
+* Country of organisation = [Russian Federation](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWMFIi)
+
+---
+
+* Name = Zebrocy
+* Full aliases = [PETROVITE](https://github.com/Panopticon-Project/panopticon-Zebrocy/blob/master
+
+---
+
+* Name = APT29
+* Full Aliases = [Cozy Bear, The Dukes](https://www.welivesecurity.com/2019/10/17/operation-ghost-dukes-never-left/), [Nobelium](https://www.bleepingcomputer.com/news/security/microsoft-russian-hackers-used-4-new-malware-in-usaid-phishing/), [UNC2452](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWMFIi), [YTTRIUM, IRON HEMLOCK, Dark Halo, NOBELIUM, StellarParticle, UNC2452](https://www.secureworks.com/research/threat-profiles), [Blue Kitsune](https://www.pwc.co.uk/issues/cyber-security-services/insights/cleaning-up-after-wellmess.html), [CozyDuke, NobleBaron](https://www.cisa.gov/uscert/ncas/alerts/aa22-110a), [HotCousin](https://securelist.com/apt-trends-report-q3-2022/107787/)
+* Organisation = [Foreign Intelligence Service/SVR](https://www.cisa.gov/uscert/ncas/alerts/aa22-110a)
+* Country of organisation = [Russian Federation](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWMFIi)
+
+---
+
+* Name = APT30
+* Organisation = [China's People’s Liberation Army Unit 78020](https://cdn2.hubspot.net/hubfs/454298/Project_CAMERASHY_ThreatConnect_Copyright_2015.pdf)
+* Country of organisation = [China](https://cdn2.hubspot.net/hubfs/454298/Project_CAMERASHY_ThreatConnect_Copyright_2015.pdf)
+
+---
+
+* Name = APT31
+* Country of organisation = [China](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWMFIi)
+
+---
+
+* Name = APT32
+* Aliases = [WOODLAWN (SCWX CTU), TIN WOODLAWN](https://www.secureworks.com/research/threat-profiles/tin-woodlawn), [BISMUTH](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWMFIi) 
+
+---
+
+* Name = APT33
+* Full Aliases = [Elfin, Refined Kitten, Magnallium, Holmium](https://brica.de/alerts/alert/public/1277530/hunting-apt33-campaign-infrastructure/), [COBALT TRINITY, TA451](https://www.secureworks.com/research/threat-profiles)
+* Organisation = [Islamic Revolutionary Guard Corps](https://www.hhs.gov/sites/default/files/iranian-threat-actors-and-healthcare.pdf)
+* Country of organisation = [Islamic Republic of Iran](https://www.hhs.gov/sites/default/files/iranian-threat-actors-and-healthcare.pdf)
+
+---
+
+* Name = APT34
+
+---
+
+* Name = CHRYSENE
+
+---
+
+* Name = APT35
+* Subgroups = [DEV-0270/Nemesis Kitten](https://www.microsoft.com/security/blog/2022/09/07/profiling-dev-0270-phosphorus-ransomware-operations/)
```

Comment:
As stated in another comment, I'm working off of comma separated values. I will therefore consider `DEV-0270/Nemesis Kitten` as one group name

Reply:
I've removed the /.

Reply:
Thanks!